alf/summary/render.py

```diff
@@ -175,7 +181,8 @@ def pack_image_nest(cls, imgs):
         # first get all images' sizes (w,h)
         sizes = [(i.shape[1], i.shape[0]) for i in imgs]
         # call rpack for an approximate solution: [(x,y),...] positions
-        positions = rpack.pack(sizes)
+        positions = rpack.pack(
+            sizes, max_width=max_width, max_height=max_height)
```

Comment:
What if there is no such a solution given the max_width and max_height? Will that crash the function?

Reply:
> In that case, a `rpack.PackingImpossibleError` will be thrown. Do you prefer having the user handle the error or catching this and rerunning without limits perhaps?

I think the second option is better.

Reply:
Changed the behavior to catch the error and rerun with constraints dropped. A warning is also printed when this occurs. PTAL.